app/assets/javascripts/jcrop_data_objects.js

```diff
@@ -48,20 +48,47 @@ $(function(){
     });
   }
 
-  function updatePreviewForm(c)
+  function updatePreviewForm(coords)
   {
-    $crop_form.children('[name="x"]').val(c.x);
-    $crop_form.children('[name="y"]').val(c.y);
-    $crop_form.children('[name="w"]').val(c.w);
-    $crop_form.children('[name="h"]').val(c.h);
+    var w = $targetImage[0].naturalWidth;
+    var h = $targetImage[0].naturalHeight;
+    if (typeof w == 'undefined') {
+      // IE 6/7/8 doesn't define naturalWidth etc, 
+      // so load up a hidden copy to get the original widths
+	  var newImg = new Image();
+	  newImg.onload = function() {fillForm(this.width, this.height, coords);};
+	  newImg.src = $targetImage[0].src;
+    } else {
+      fillForm(w, h, coords);
+    }
   }
-
+  
+  function fillForm(w,h,c)
+  {
+    //EoL specific: express crop as % not px, as large images may be shrunk
+    // Offsets are from the 580 x 360 image. However, if they are wider than 
+    // 540px, the EoL CSS scales the image proportionally to fit into a max 
+    // width of 540. %ages thus need scaling to match the image dimensions
+    var scaleFactor = 1;
+    if((w / h) < ( 540 / 360 ))
+    {
+      //smaller width, so scaling only happens if height exceeds max
+      if(h > 360) scale_factor = h / 360;
```

Comment:
Identifier 'scale_factor' is not in camel case.<br>Expected '{' and instead saw 'scale_factor'.<br>'scale_factor' is not defined.